README.adoc

```diff
@@ -445,23 +679,27 @@ Tests run: 4, Failures: 0, Errors: 0, Skipped: 0
 
 == Tearing down the environment
 
-When you no longer need your deployed microservices, you can delete all {kube} resources by running the `kubectl delete` command:
+When you no longer need your deployed microservices, 
+you can delete all {kube} resources by running the `kubectl delete` command:
 [role='command']
 ```
 kubectl delete -f kubernetes.yaml
 ```
 
+// Static guide only instruction
+ifndef::cloud-hosted[]
 [role='command']
 include::{common-includes}/kube-minikube-teardown.adoc[]
-
+endif::[]
 
 // =================================================================================================
 // finish
 // =================================================================================================
 
 == Great work! You're done!
 
-You have just deployed two microservices running in Open Liberty to {kube}. You then scaled a microservice and ran integration tests against miroservices that are running in a {kube} cluster.
+You have just deployed two microservices running in Open Liberty to {kube}. 
```

Comment:
You have just deployed two microservices that are running in Open Liberty to {kube}.